packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -542,7 +542,7 @@ const defaultChunkSizeInBytes = 204800;
  * of the current system, we should close the summarizer and let it recover.
  * This delay's goal is to prevent tight restart loops
  */
-const defaultCloseSummarizerDelayMs = 10000; // 10 seconds
+const defaultCloseSummarizerDelayMs = 5000; // 10 seconds
```

Comment:
Adjusted the comment, but honestly, I think we can remove it; the variable name already has `Ms` so I think it's clear enough and the comment doesn't add too much.
```suggestion
const defaultCloseSummarizerDelayMs = 5000; // 5 seconds
```